frontend/src/services/Api.ts

```diff
@@ -102,8 +102,8 @@ const loginHeaders = config.login.headers;
 
 /**  Helpers to Requests */
 
-const getHeaders = (): Partial<AxiosHeaders> => {
-  if (apiProxy) {
+const getHeaders = (urlEncoded?: boolean): Partial<AxiosHeaders> => {
+  if (apiProxy || urlEncoded) {
```

Comment:
It looks like this leaves off `...loginHeaders`. Should those be included for both content types?

Reply:
I've thought that loginHeaders contain the login information (Once logged in), but let me verify

Reply:
Updated

Reply:
I'm wondering if OSSMC should also have those login headers? @ferhoyos?

Reply:
I don't think so, as OSSMC leverages of Openshift login. Where `'X-Auth-Type-Kiali-UI': '1'` is used?

Said that, including loginHeaders probably do not affect OSSMC. Maybe you can simplify the code:
```
    if (apiProxy || urlEncoded) {
      return { 'Content-Type': 'application/x-www-form-urlencoded', ...loginHeaders };
    } else {
      return { 'Content-Type': 'application/json', ...loginHeaders };
   }
```